src/app/docs/button/button-docs.component.html

```diff
@@ -0,0 +1,196 @@
+            </p>
+            <p>
+              Secondary buttons trigger less important or less frequently used actions. Since secondary buttons are less important than primary buttons, SKY UX reduces the visual weight to make them less attention-drawing than the primary buttons.
+            </p>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Code
+            </p>
+            <p>
+              <sky-code>&lt;button type=&quot;button&quot; class=&quot;sky-btn sky-btn-default&quot;&gt;Secondary&lt;/button&gt;</sky-code>
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            <button type="button" class="sky-btn sky-btn-link">Link</button>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Usage
+            </p>
+            <p>
+              Link buttons trigger actions that take users to other pages or actions that end tasks or exit modals (such as <strong>Cancel</strong>).
+            </p>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Code
+            </p>
+            <p>
+              <sky-code>&lt;button type=&quot;button&quot; class=&quot;sky-btn sky-btn-link&quot;&gt;Link button&lt;/button&gt;</sky-code>
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            <button type="button" class="sky-btn sky-btn-link-inline">Inline link</button>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Usage
+            </p>
+            <p>
+              Inline link buttons trigger the same type of actions as link buttons, but the styling is altered to fit inline.
+            </p>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Code
+            </p>
+            <p>
+              <sky-code>&lt;button type=&quot;button&quot; class=&quot;sky-btn sky-btn-link-inline&quot;&gt;Inline link&lt;/button&gt;</sky-code>
+            </p>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            <div style="margin-bottom: 10px;">
+              <button type="button" class="sky-btn sky-btn-primary sky-btn-disabled">Primary</button>
+            </div>
+            <div style="margin-bottom: 10px;">
+              <button type="button" class="sky-btn sky-btn-default sky-btn-disabled">Secondary</button>
+            </div>
+            <div style="margin-bottom: 10px;">
+              <button type="button" class="sky-btn sky-btn-link sky-btn-disabled">Link</button>
+            </div>
+          </td>
+          <td class="sky-docs-table-cell">
+            <p>
+              Usage
+            </p>
+            <p>
+              The disable class disables all button types. The most common use case is to disable actions until users complete prerequisite tasks.
```

Comment:
I noticed we don't have any titles for each of these rows. The user has to read the button images to understand what type of button we're talking about. That's not a big deal except the last button isn't titled "disabled". Should we update the image or add a header to make these all consistent?

Reply:
I'm not sure I follow your suggestion to "update the image or add a header," but I wonder if the answer here is to change the button text for all 4 items in the last row to just `Disabled` (and not worry about the rest of the button text). What do you think, @Blackbaud-AlexKingman?

Reply:
Yep, that works for me!